packages/migrate/src/commands/DbPull.ts

```diff
@@ -1,6 +1,7 @@
-import type { Command, IntrospectionSchemaVersion, IntrospectionWarnings } from '@prisma/sdk'
 import {
   arg,
+  checkUnsupportedDataProxy,
+  Command,
```

Comment:
What is that change from `import type {Command ...` to `import { Command ...` that we have in multiple places? Relevant?

Reply:
It's the linting + type imports that stopped working (not a big deal but something to attend).